tests/marray/marray_operators.h

```diff
@@ -224,6 +224,52 @@ class run_binary_sequence_scalar {
   }
 };
 
+template <typename InitSeqForRhs, std::size_t seq_el_num>
+inline constexpr bool init_seq_contains_too_big_values_for_shift_op(
+    std::size_t max_shift_wo_undef_behavior) {
+  return (std::is_same_v<InitSeqForRhs, seq_inc> ||
+          std::is_same_v<InitSeqForRhs, seq_dec<seq_el_num>>)&&seq_el_num >
+         max_shift_wo_undef_behavior;
```

Comment:
@keryell, will it be acceptable to disable automatic formatting for this template [with comments](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code) and format it manually?

Reply:
As you want.

Reply:
> As you want.

I prefer to not fight with clang-format.
@vladimirkhashev, do you think it's necessary to re-format to make it more readable or we can merge automatic formatting?

Reply:
@bader, in my opinion we could leave it as it is.

Reply:
Yes, at some point `clang-format` will make some progress (if not done already) and will fix it, whereas if we add a comment forbidding `clang-format` here it will never happen.